pkg/volume/azure_file/BUILD

```diff
@@ -1,10 +1,4 @@
-package(default_visibility = ["//visibility:public"])
-
-load(
-    "@io_bazel_rules_go//go:def.bzl",
-    "go_library",
-    "go_test",
-)
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
```

Comment:
this change was slightly surprising... did you run hack/update-bazel.sh after rebasing?

Reply:
That's where it came from! I can rebase 1 more time locally and re-run it. If there are any new changes I'll push it.

But yeah, after re-basing, it did touch imports in `pkg/volume/azure_file`, so I removed that `BUILD` file and regenerated it...

Reply:
yeah, just re-ran and it produces the same output... No idea why it suddenly wants to stuff everything on one line.